lib/cisco_node_utils/ntp_server.rb

```diff
@@ -66,20 +64,41 @@ def ==(other)
 
     def create
       config_set('ntp_server', 'server', state: '', ip: @ntpserver_id,
-                  prefer: @ntpserver_prefer ? 'prefer' : '')
+                  prefer: @prefer == 'true' ? 'prefer' : '',
+                  vrf: @vrf ? "use-vrf #{@vrf}" : '',
+                  key: @key ? "key #{@key}" : '',
+                  minpoll: @minpoll ? "minpoll #{@minpoll}" : '',
+                  maxpoll: @maxpoll ? "maxpoll #{@maxpoll}" : '')
     end
 
     def destroy
```

Comment:
Ah, that's because the other properties need to get passed in as empty string.  I suppose this is fine.